src/main/java/com/conveyal/taui/controllers/ProjectController.java

```diff
@@ -85,6 +92,21 @@ public static Project update(Request req, Response res) throws IOException {
 
                     // Matched up the phased entries and timetables
                     if (modification.getType().equals(AddTripPattern.type)) {
+                        if (bundlesAreNotEqual) {
+                            // Remove references to real stops in the old bundle
```

Comment:
To make sure I understand this correctly, segment.geometry, stopAtStart, and stopAtEnd are unchanged.  So this just removes the feed-scoped stopIds, which so the stop will be interpreted as newly created.